eng/pipelines/templates/jobs/perf.yml

```diff
@@ -0,0 +1,111 @@
+parameters:
+  - name: serviceDirectory
+    type: string
+    default: ''
+  - name: services
+    type: string
+    default: ''
+  - name: packageVersions
+    type: string
+    default: '.*'
+  - name: tests
+    type: string
+    default: ''
+  - name: arguments
+    type: string
+    default: '' 
+  - name: iterations
+    type: number
+    default: '5'
+  - name: envVars
+    type: object
+    default: {}
+  - name: additionalArguments
+    type: string
+    default: ''
+
+variables:
+  - name: NugetSecurityAnalysisWarningLevel
+    value: 'none'
+  - name: skipComponentGovernanceDetection
+    value: 'true'
+  - name: Pool
+    value: 'azsdk-pool-mms-ubuntu-2004-perf'
+  - name: OSVmImage
+    value: 'MMSUbuntu20.04'
+
+resources:
+  repositories:
+  - repository: azure-sdk-tools
+    type: github
+    endpoint: Azure
+    name: Azure/azure-sdk-tools
```

Comment:
We should include a ref, usually a release tag.

Reply:
For now we are using main.  I will add an explicit reference to main.  I know this means the build is unstable, but while the tooling is still being actively developed, I don't want to require PRs to every language every time we update the tooling.